README.md

```diff
@@ -120,6 +120,23 @@ Script ini membutuhkan file zip dari submission setiap kelas. Didapatkan dengan
 
    ![image-20220124144037000](docs/image-20220124144037000.png)
 
+5. (Opsional) Karena link Moss hanya valid selama 3 hari, dapat dilakukan scraping untuk menyimpan HTML hasil laporan di device lokal. Untuk itu, dapat digunakan script `scrape_report.sh` dengan langkah-langkah seperti berikut
```

Comment:
I believe it's better to put this guide under the "Menggunakan Script" section. Split the section with, for example, two subsections of "moss_tp.py" and "scrape_report.sh", then put this guide under the latter.

Reply:
Change "seperti berikut" to "sebagai berikut"